public/app/features/dashboard-scene/scene/DashboardSceneUrlSync.ts

```diff
@@ -28,7 +28,7 @@ export class DashboardSceneUrlSync implements SceneObjectUrlSyncHandler {
   constructor(private _scene: DashboardScene) {}
 
   getKeys(): string[] {
-    return ['inspect', 'viewPanel', 'editPanel', 'editview', 'autofitpanels'];
+    return ['inspect', 'viewPanel', 'editPanel', 'editview', 'autofitpanels', 'isNewPanel'];
```

Comment:
why do we need isNewPanel in url state?

Reply:
> I used the URL to move the state between scene components. When we create the editor, the object stores in the state if the panel is newly created. To get the "isNew" state, we set that parameter from the CTAs that set the editPanel into the URL.

When adding the visualization, we add the panel calling a function from DashboardScene. Right after, we change the URL to set that key as `editPanel`. We do this from the "Add -> Visualization when editing" and "Add visualization" from the empty state. To get that state in the DashboardScene when getting a new value for  `editPanel` we don't have a way to know if it is editing a new panel unless we set it through the URL when changing. 

I didn't have any other idea how to create PanelEditor with this state if it is not using the URL state. Any better idea?